packages/dashboard/src/pages/CreateMaskWallet/AddDeriveWallet/index.tsx

```diff
@@ -83,7 +82,7 @@ const AddDeriveWallet = memo(function AddDeriveWallet() {
 
     const { loading, value: dataSource } = useAsync(async () => {
         if (mnemonic) {
-            const unDeriveWallets = Array.from(indexes)
+            const unDeriveWallets = Array.from(indexes.current)
```

Comment:
Do not mix selected status into `dataSource`, maintain a `selectedAddresses` list and pass to the table instead. Thinking of a multiple `<select />`

Reply:
We need `pathIndex` to generate wallet.

```
PluginServices.Wallet.recoverWalletFromMnemonic(
    `${walletName}${pathIndex}`,
    mnemonic,
    `${HD_PATH_WITHOUT_INDEX_ETHEREUM}/${pathIndex}`,
),
```

Reply:
`pathIndex !== index`